packages/snap-preact-components/src/components/Templates/Search/Search.tsx

```diff
@@ -158,12 +171,18 @@ export const Search = observer((properties: SearchProps): JSX.Element => {
 		styling.css.push(styleScript(props));
 	}
 
+	const mobileMediaQuery = `(max-width: ${slideOutToggleWidth})`;
+	const isMobile = useMediaQuery(mobileMediaQuery);
+
 	return (
 		<ThemeProvider theme={properties.theme || {}}>
 			<CacheProvider>
 				<div {...styling} className={classnames('ss__search', className)}>
-					{!hideSidebar && <Sidebar {...subProps.Sidebar} controller={controller} />}
+					<div className="ss__search__sidebar-wrapper">
+						{!hideSidebar && !isMobile && <Sidebar {...subProps.Sidebar} controller={controller} />}
 
+						<MobileSidebar controller={controller} {...subProps.MobileSidebar} />
```

Comment:
The positioning here is awkward and could be better - but something we can adjust later.

<img width="504" alt="image" src="https://github.com/searchspring/snap/assets/5060400/5a37c84a-e91e-4955-a7e2-53d39586d7bd">